chainweb.cabal

```
@@ -351,7 +351,7 @@ library
         , monad-control >= 1.0
         , mtl >= 2.2
         , mwc-random >= 0.13
-        , mwc-probability >= 2.0 && <2.2
+        , mwc-probability >= 2.0 && <2.3
```

This bound trivial, so let's drop it.

```suggestion
        , mwc-probability >= 2.0
```